css/unsupported-browser/_unsupported-desktop-browser.scss

```diff
@@ -1,132 +1,39 @@
-.supported-browser {
-    color: #929391;
-    display: inline-block;
-    font-size: 20px;
-    margin: 1em 7px;
-    vertical-align: middle;
-    width: 138px;
-
-    &__button {
-        background-color: #62c82a;
-        border: 1px solid #3c8117;
-        border-radius: 10px;
-        color: #FFFFFF;
-        font-size: 12px;
-        height: 26px;
-        margin: 15px auto 0px auto;
-        padding-top: 13px;
-        text-align: center;
-        width: 115px;
-    }
-
-    &__link {
-        color: #087dba;
-        text-decoration: none;
-
-        &:hover {
-            text-decoration: none;
-        }
-
-        &:active {
-            text-decoration: none;
-        }
-
-        &:focus {
-            text-decoration: none;
-        }
-    }
-
-    &-list
-    {
-        margin: 0 auto;
-    }
-
-    &__logo {
-        margin: 20px auto 0px auto;
-
-        &_chrome {
-            background-image: url('../../images/chrome.png');
-            height: 78px;
-            width: 78px;
-        }
-
-        &_chromium {
-            background-image: url('../../images/chromium.png');
-            height: 78px;
-            width: 77px;
-        }
-
-        &_firefox {
-            background-image: url('../../images/firefox.png');
-            height: 80px;
-            width: 86px;
-        }
-
-        &_opera {
-            background-image: url('../../images/opera.png');
-            height: 78px;
-            width: 73px;
-        }
+.unsupported-desktop-browser {
+    @include absoluteAligning();
 
-        &_ie {
-            background-image: url('../../images/ie.png');
-            height: 78px;
-            width: 80px;
-        }
+    display: block;
+    text-align: center;
 
-        &_safari {
-            background-image: url('../../images/safari.png');
-            height: 79px;
-            width: 78px;
-        }
+    &__title {
+        color: #fff;
+        font-weight: 300;
+        font-size: 24px;
+        letter-spacing: 1px;
     }
 
-    &__text
-    {
-        line-height: 1.2em;
+    &__description {
+        margin-top: 16px;
+        color: rgba(255, 255, 255, 0.7);
```

Comment:
Done! I created variables for colors and font-sizes in `_variables.scss` because value for all themes are the same.